cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -1128,6 +1144,29 @@ impl PrettyPrint for Inst {
                 format!("{} {}, {}, {}", ljustify(op.to_string()), src1, src2, dst)
             }
 
+            Inst::XmmRmRVex {
+                op,
+                src1,
+                src2,
+                src3,
+                dst,
+                ..
+            } => {
+                let src3 = src3.pretty_print(8, allocs);
+                let src2 = pretty_print_reg(src2.to_reg(), 8, allocs);
+                let src1 = pretty_print_reg(src1.to_reg(), 8, allocs);
+                let dst = pretty_print_reg(dst.to_reg().to_reg(), 8, allocs);
```

Comment:
The order of these uses of `allocs` needs to match the order in which registers are given to the `OperandCollector` in `get_operands` below. (I know, it's a terrible footgun... we hope to make this a bit more ergonomic later!)

Reply:
Thanks! I'm still not too comfortable with this.